### ISSUE TITLE
Displaying more items for the user to buy.

### ISSUE THREAD

**Issue by omarf72** [omarf72/CSCchallenge1#11]
I think there should be variety of items that a user can buy. I think this will make it more realistic.

**Comment by asgoodman224** [comment]
I agree with this because if we have too many items that a user can buy. It won't act as an actual vending machine like the ones in person.
